hawc/apps/summary/api.py

```diff
@@ -115,3 +115,15 @@ def create(self, request, *args, **kwargs):
         if not self.assessment.user_can_edit_object(request.user):
             raise exceptions.PermissionDenied()
         return super().create(request, *args, **kwargs)
+
+
+class SummaryTableViewset(AssessmentViewset):
+    assessment_filter_args = "assessment"
+    model = models.SummaryTable
+    serializer_class = serializers.SummaryTableSerializer
+
+    @action(detail=True, methods=("get",), renderer_classes=(DocxRenderer,))
```

Comment:
add `filter_backends = (InAssessmentFilter, UnpublishedFilter)` like the `VisualViewset`